liquid_node/configuration.py

```diff
@@ -486,16 +487,25 @@ def is_app_enabled(self, app_name):
 
     @classmethod
     def _validate_collection_name(self, name):
```

Comment:
First argument for `@classmethod` should be `cls` not `self`, to signify we have a class type, not an object. Please fix this in this patch.